app/assets/javascripts/employers_edit.js.erb

```diff
@@ -0,0 +1,49 @@
+function fields_have_errors() {
+  var hiring_commitment = $("#employer_commit_to_hire").val();
+  var ein = $("#employer_ein").val();
+  var num_text = '';
+  var error_text = '';
+  if (hiring_commitment && (hiring_commitment < 1)) {
+    num_text = '1 error';
+    error_text = '<li>Commit to hire must be a postive number</li>';
+  }
+  if (ein && (ein.length < 9)) {
```

Comment:
This `<` allows EIN's of length 10 to not throw an error. Should it be `!=` instead?